tests/unit/core/utils/test_compatibility.js

```diff
@@ -646,8 +720,20 @@ describe(__filename, () => {
       expect(isQuantumCompatible({ addon })).toEqual(false);
     });
 
-    it('returns `true` when add-on is compatible with one of the platforms', () => {
+    it('returns `false` for add-ons without a current version', () => {
+      const addon = createInternalAddon(createFakeAddon({
+        current_version: null,
+      }));
+
+      expect(isQuantumCompatible({ addon })).toEqual(false);
+    });
+
+    it('returns `true` when Android webextension is compatible', () => {
       const addon = createInternalAddon(createFakeAddon({
+        files: [{
+          is_webextension: true,
+          platform: OS_ANDROID,
```

Comment:
When I added this, the old code was failing because of the desktop UA string. Oops.

Reply:
😞